lib/gridfinityUtils/baseGenerator.py

```diff
@@ -199,6 +226,17 @@ def createGridfinityBase(
         patternFeature = rectangularPatternFeatures.add(patternInput)
         combineUtils.cutBody(baseBody, commonUtils.objectCollectionFromList(list(cutoutBodies) + list(patternFeature.bodies)), targetComponent)
 
+        if input.hasMagnetCutoutsTabs:
+            patternTabInput = rectangularPatternFeatures.createInput(patternTabCutoutBodies,
```

Comment:
I wonder if usage of circular pattern would make it easier to derive inputs here

Reply:
it also would be nice to make it part of the hole cutout body which could be possible with circular pattern, so that the process of making these cut is more consistent